src/styles/layout/_navbar.scss

```diff
@@ -0,0 +1,33 @@
+// Styling for the navbar
+#mainNav {
```

Comment:
No IDs ever in React. They cause all sorts of issues (use a class instead). Atomic BEM for everything in this file please.